scala/scala.bzl

```diff
@@ -369,3 +550,68 @@ def scala_repositories():
     url = "https://oss.sonatype.org/content/groups/public/org/scalatest/scalatest_2.11/2.2.6/scalatest_2.11-2.2.6.jar",
     sha256 = "f198967436a5e7a69cfd182902adcfbcb9f2e41b349e1a5c8881a2407f615962",
   )
+
+def scala_2_10_repositories():
+  native.new_http_archive(
+    name = "scala",
+    strip_prefix = "scala-2.10.6",
+    sha256 = "54adf583dae6734d66328cafa26d9fa03b8c4cf607e27b9f3915f96e9bcd2d67",
+    url = "https://downloads.lightbend.com/scala/2.10.6/scala-2.10.6.tgz",
+    build_file_content = SCALA_2_10_BUILD_FILE,
+  )
+
+def zinc_repositories():
+  native.http_file(
+    name = "zinc",
+    url = "https://databricks-mvn.s3.amazonaws.com/binaries/zinc/12-29-15/zinc?AWSAccessKeyId=AKIAJ6V3VSHTA5RSYEQA&Expires=1482970631&Signature=tAP2QKWEnte6v3DpRFtAxGNkUps%3D",
```

Comment:
These were links being used for our internal build. This is the customized version of zinc that I mentioned above. I don't believe that we actually need it anymore, so I can just pull zinc from typesafe or github. I'll just have to update the PR to be able to work with standard zinc instead of my fork